integration/test_project_config_mgt.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests for firebase_admin.project_config_mgt module."""
+
+import pytest
+
+from firebase_admin import project_config_mgt
+
+
+@pytest.fixture(scope='module')
+def sample_mfa_config():
+    mfa_config = {
+        'state': 'ENABLED',
+        'factorIds': ['PHONE_SMS'],
+        'providerConfigs': [
+            {
+                'state': 'ENABLED',
+                'totpProviderConfig': {
+                    'adjacentIntervals': 5
```

Comment:
If you can put some of these strings in to constant, it will be easier for the test cases to refer back to them and easier compare the value. And do this for all tests where string and numbers shows up repeatedly?